components/suggest/src/suggestion.rs

```diff
@@ -96,13 +96,26 @@ pub enum Suggestion {
         icon: Option<Vec<u8>>,
         icon_mimetype: Option<String>,
         score: f64,
+        // Details about the FTS match.  For performance reasons, this is only calculated for the
+        // result with the highest score.  That's the only one that will be shown to the user and
```

Comment:
Turbo nit: I would phrase this more as assumption the component is making about how the consumer will use these suggestions, e.g., "we assume consumers will only show the highest-scoring suggestion."